ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -109,19 +87,16 @@ Item {
             Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
             Layout.topMargin: Style.current.bigPadding
             StatusSmartIdenticon {
-                anchors.left: parent.left
                 id: userImage
                 objectName: "welcomeScreenUserProfileImage"
+                anchors.left: parent.left
                 asset.width: 86
                 asset.height: 86
                 asset.letterSize: 32
-                asset.color: Utils.colorForPubkey(root.pubKey)
+                asset.color: Utils.colorForColorId(0)
```

Comment:
Why `0` here?

Reply:
I will update PR description with explanation when finished.

Reply:
Added a comment in the code.